src/lib/layout/unauthenticated.svelte

```diff
@@ -291,6 +291,13 @@
         }
     }
 
+    /* mobile only: remove top padding toeliminate extra space */
+    @media #{devices.$break1} {
+        .side-default {
+            padding-block-start: 0;
+        }
+    }
```

Comment:
we don't need this. instead add the `side-default` if `!$isSmallViewport` conditionally.